cfme/tests/configure/test_access_control.py

```diff
@@ -2979,3 +3031,59 @@ def test_select_edit_group(appliance, request):
         view.toolbar.configuration.open()
         assert view.toolbar.configuration.item_enabled(
             "Edit the selected Group"), '"Edit the selected Group" button is disabled'
+
+
+@pytest.mark.customer_scenario
+@test_requirements.rbac
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1730066])
```

Comment:
Use the `automates` keyword instead of `coverage` for automated tests.

Reply:
Done